### ISSUE TITLE
'tools/run_in_bg.c:87: error: undefined reference to 'stderr'' with Android NDK 15b

### ISSUE THREAD

**Issue by fhahn** [DynamoRIO/dynamorio#2556]
Building DyanmoRIO with the latest Android NDK (15b) fails with the following error

```
dynamorio/tools/run_in_bg.c:87: error: undefined reference to 'stderr'
dynamorio/tools/run_in_bg.c:157: error: undefined reference to 'stderr'
dynamorio/tools/run_in_bg.c:157: error: undefined reference to 'stdout'
```
Apparently the global variables for stderr and stdout have been removed from Bionic (https://stackoverflow.com/questions/39322852/android-ndk-undefined-reference-to-stderr)

Xref #1927, which describes another issue when building with recent NDK versions.

**Comment by derekbruening** [comment]
@mxmssh you hit this recently right?

**Comment by mxmssh** [comment]
Yes, I had such problem with a new version of Android NDK. I just installed older version (14b) and it solves the issue. 
https://groups.google.com/forum/#!topic/drmemory-users/BnGpOmopmZc
